sites/partners/src/pages/index.tsx

```diff
@@ -30,7 +32,7 @@ class formatWaitlistStatus {
   text: HTMLSpanElement
 
   init({ data }) {
-    const isWaitlistOpen = data.waitlistCurrentSize < data.waitlistMaxSize
```

Comment:
This was just incorrect. Core listings don't have access to these fields anymore.